docs/rfc/README.md

```diff
@@ -0,0 +1,39 @@
+---
+order: 1
+parent:
+  order: false
+---
+
+# Requests for Comments
+
+A Request for Comments (RFC) is a record of discussion on an open-ended topic
+related to the design and implementation of Tendermint Core, for which no
```

Comment:
nit - doesn't have to just be Tendermint

Reply:
nice catch